.travis.yml

```diff
@@ -26,7 +26,7 @@ install:
 
 script:
     - if [[ $TEST_TARGET == 'default' ]]; then
-        python setup.py test -a "--verbose";
+        python setup.py test -a '--verbose --cov=pyugrid';
```

Comment:
so what happens if you don't have 100% coverage? I'd like to see the report, but we don't want that to be the equivalent of a test failure.